specification/changeanalysis/resource-manager/readme.python.md

```diff
@@ -3,19 +3,16 @@
 These settings apply only when `--python` is specified on the command line.
 Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
 
-```yaml $(python)
-python:
-  azure-arm: true
-  license-header: MICROSOFT_MIT_NO_VERSION
-  payload-flattening-threshold: 2
-  namespace: Microsoft.ChangeAnalysis
-  package-name: changeanalysis
-  package-version: 2020-04-01-preview
-  clear-output-folder: true
+```yaml $(python) && $(track2)
+azure-arm: true
+license-header: MICROSOFT_MIT_NO_VERSION
+namespace: Microsoft.ChangeAnalysis
+package-name: changeanalysis
+package-version: 1.0.0b1
+clear-output-folder: true
 ```
 
-```yaml $(python)
-python:
-  no-namespace-folders: true
-  output-folder: $(python-sdks-folder)/azure-mgmt/changeanalysis
+```yaml $(python) && $(track2)
+no-namespace-folders: true
+output-folder: $(python-sdks-folder)/azure-mgmt/changeanalysis
```

Comment:
It is obvious that the output-folder is not right. New service for track2-only config could reference : https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/samplefiles/readme.python.md